cdap-ui/app/cdap/components/PipelineList/PipelineList.scss

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import '~styles/colors.scss';
+
+$pipeline-view-header: 55px;
+$pipeline-bg-color: $grey-08;
+$view-header-bg-color: white;
+$cog-icon-color: $grey-04;
+$cog-icon-hover-color: $blue-04;
+$popover-width: 100px;
+$delete-color: $red-02;
+$disabled-color: $grey-04;
+
+.pipeline-list {
+  // substracting the height of global header and footer
+  height: calc(100vh - 50px - 53px);
+  background-color: $pipeline-bg-color;
+
+  .cask-resourcecenter-button { top: 75px; }
+
+  .pipeline-list-content {
+    height: calc(100% - #{$pipeline-view-header});
+    overflow: auto;
+    padding-bottom: 10px;
+  }
+
+  .view-header {
+    background-color: $view-header-bg-color;
+    height: $pipeline-view-header;
+    padding-left: 45px;
+    line-height: $pipeline-view-header;
+    margin-bottom: 0;
+    font-size: 18px;
+
+    .option { cursor: pointer; }
+
+    .active {
+      font-weight: 600;
+      text-decoration: underline;
+    }
+
+    .separator {
+      margin-left: 15px;
+      margin-right: 15px;
+    }
+  }
+
+  .icon-svg.icon-cog-empty {
+    fill: none;
+    stroke: $cog-icon-color;
+    font-size: 23px;
+    cursor: pointer;
+
+    &:hover,
+    &.active {
+      stroke: $cog-icon-hover-color;
+    }
+  }
+}
+
+.pipeline-list-popover {
+  display: inline-block;
+
+  .popper {
+    width: $popover-width;
+    min-width: $popover-width;
+    padding: 0;
+
+    hr { margin: 5px; }
```

Comment:
@elfenheart Seems adhoc. Right now because of this change the same popover looks different from the one in pipeline detailed view.

We seem to be using the same popover in detailed view/listview and in drafts list. Could extract this component and reuse it in other places?

Reply:
that's a good point, will extract this to a separate component